ui/src/components/terminal/TerminalDialog.vue

```diff
@@ -178,6 +178,13 @@ export default {
   },
 
   computed: {
+    xtermDimensions() {
```

Comment:
This is used only on the web terminal, correct? In this case, it might be better to call it as `webTermDimensions`